pkg/scheduler/scheduler.go

```diff
@@ -287,7 +287,7 @@ func (sched *Scheduler) schedule(pod *v1.Pod) (core.ScheduleResult, error) {
 		sched.recordSchedulingFailure(pod, err, v1.PodReasonUnschedulable, err.Error())
 		return core.ScheduleResult{}, err
 	}
-	return result, err
+	return result, nil
```

Comment:
that is not ab bug, err is going to be nil here anyways, right?

Reply:
/remove-kind bug
/kind cleanup
/lgtm